test/bob-marley-spec.js

```diff
@@ -0,0 +1,15 @@
+require('should');
+import fs from 'fs';
+import parse from '../index';
+
+describe.only("Should Parse Bob Marley's Information", () => {
```

Comment:
Whoops. I just realized you left the `.only`. I'll fix it

Reply:
Ouch! Sorry fro that!